src/components/layout/test/Panel-test.js

```diff
@@ -0,0 +1,37 @@
+import { mount } from 'enzyme';
+
+import { testCompositeComponent } from '../../test/common-tests';
+
+import Panel from '../Panel';
+import { EditIcon } from '../../icons';
+
+const createComponent = (Component, props = {}) => {
+  return mount(<Component {...props}>This is child content</Component>);
+};
+
+describe('Panel', () => {
+  testCompositeComponent(Panel, 'Panel');
+
+  // This is an auto-generated test. Expand or delete.
```

Comment:
Does anything need to be done with this TODO?

Reply:
This is part of the auto-generated test file from the `plop` scaffolding I'm using...hmmm...in this case I did the "expanding" as indicated but neglected to remove the comment...hmmmm...I'll think about how not to overlook this.